### DISCUSSION TITLE
Failed to launch the browser process!

### DISCUSSION THREAD

**Discussion by MikePageDev** [spatie/browsershot#501] [Q&A]
I am trying to create and download a PDF from HTML in a laravel project using verumconsilium/laravel-browsershot.
```
return PDF::loadView('pdf.certificates.default', $data)
            ->download();
```
however, I am getting the following error

`Error: Failed to launch the browser process! spawn /usr/bin/chromium ENOENT`

I am using devilbox as my development environment. Which I think maybe the problem? Something to do with the path? Sorry I don't use node so am a bit lost

**Comment by ahmdqader** [comment]
I'm facing the same problem and I need help, I'm also using devilbox, if you resolved it please reply with the solution
